.github/workflows/touchdown-pipeline/touchdown-build.yml

```diff
@@ -23,73 +23,82 @@ extends:
                 inputs:
-                  environment: 'PRODEXT'
-                  teamId: '35669'
-                  authType: 'OAuth'
-                  authId: 'fa441387-cab4-4f61-969d-fd23388fc7b4'
-                  authKey: $(AUTH_KEY)
-                  isPreview: false
-                  resourceFilePath: 'packages/react-components/src/localization/locales/en-US/strings.json'
-                  outputDirectoryRoot: 'packages/react-components/src/localization/locales'
-                  cultureMappingType: 'None'
-                  gitAction: 'COMMIT'
-                displayName: 'Call Touchdown build task for components and commit changes'
-                condition: eq(variables['commit'], 'true')
+                  targetType: 'inline'
+                  script: |
+                    # ------------------------------------------------------------------------------------------------
+                    # - Script to push changes
+                    # ------------------------------------------------------------------------------------------------
 
-              - task: TouchdownBuildTask@1
-                inputs:
-                  environment: 'PRODEXT'
-                  teamId: '35669'
-                  authType: 'OAuth'
-                  authId: 'fa441387-cab4-4f61-969d-fd23388fc7b4'
-                  authKey: $(AUTH_KEY)
-                  isPreview: false
-                  resourceFilePath: 'packages/react-composites/src/composites/localization/locales/en-US/strings.json'
-                  outputDirectoryRoot: 'packages/react-composites/src/composites/localization/locales'
-                  cultureMappingType: 'None'
-                  gitAction: 'COMMIT'
-                displayName: 'Call Touchdown build task for composites and commit changes'
-                condition: eq(variables['commit'], 'true')
+                    $teamProject = $env:SYSTEM_TEAMPROJECT
+                    $repoName= $env:Build_Repository_Name
+                    $pipelineName = $env:Build_DefinitionName
+
+                    Write-host "Azure DevOps organization: $env:System_TeamFoundationCollectionUri"
+                    Write-host "Team Project: $teamProject"
+                    Write-host "Repo Name: $repoName"
+                    Write-host "Source Branch: $(Build.SourceBranch)"
+
+
+                    if ("$(git status)".Contains("nothing to commit")) {
+                        Write-Host "Nothing to commit, exiting immediately"
+                        Exit
+                    }
+
+                    #****************************************
+                    #************Configure GIT***************
+                    #****************************************
+                    Write-Host "##[group] Configure GIT"
+
+                    #Setting Global Git Username and Email. NOTE: Branch will be created as this user
+                    git config --global user.email "79475487+mgamis-msft@users.noreply.github.com"

```

Comment:
Can we use the generic github actions bot details that we use in other actions:
```
# User id of github actions bot. See https://api.github.com/users/better-informatics%5Bbot%5D
        run: |
          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
          git config user.name "github-actions[bot]"
```

Reply:
Good catch! My bad I was testing my PAT token to create a PR. Did not work. I will try fine-grained PAT tokens on this bot account and see if that works

Reply:
Does this use a PAT token in this PR?  I don't see one referenced in the yml file

Reply:
No this does not use one. But I tried to use one while testing PR creation.